api.py

```diff
@@ -912,6 +912,39 @@ def create_asset(name, content, mime_type, **kwargs):
         **kwargs)
 
 
+def modify_asset_content(resource_id, content, mime_type, **kwargs):
+    return modify_resource_content(resource_id, content, mime_type, **kwargs)
+
+
+def modify_resource_json(resource_id, content, **kwargs):
+    return modify_resource_content(resource_id, content, 'application/json', **kwargs)
+
+
+def modify_resource_content(resource_id, content, mime_type, **kwargs):
+    if mime_type == 'application/json':
+        print "JSON encoding content for {}".format(resource_id)
+        content = json.dumps(content)
+    elif not (mime_type.startswith('text') or mime_type == 'application/json'):
+        if not is_base64_encoded(content):
+            print "base64 encoding content for {}".format(resource_id)
+            content = base64.b64encode(content)
+
+    return _modify_resource_content(resource_id, content, **kwargs)
+
+
+def _modify_resource_content(resource_id, content, **kwargs):
+    # payload = [{
+    #    'path': '/content',
+    #    'value': content,
+    #    'op': 'add'
```

Comment:
Note that the fix for this has been merged into dev.  So you can now correctly use replace.

Reply:
Not until it gets to dhl-prd

Reply:
Got it.